data/json/items/migration.json

```diff
@@ -5,6 +5,11 @@
     "replace": "adv_UPS_off"
   },
   {
+    "id": "battery_truck",
+    "type": "MIGRATION",
+    "replace": "battery_car"
```

Comment:
Is that per-se a problem beyond once you remove one of the doubled up batteries you can't reinstall it? Eventually I want to support multiple parts on a vehicle tile (#17062).